solidity/random-beacon/tasks/ensure-eth-balance.ts

```diff
@@ -0,0 +1,59 @@
+import { task, types } from "hardhat/config"
+
+import { parseValue, TASK_SEND_ETH } from "./send-eth"
+
+const TASK_ENSURE_ETH_BALANCE = "ensure-eth-balance"
+
+task(
+  TASK_ENSURE_ETH_BALANCE,
+  "Ensure addresses hold a minimum ether balance, top-up if needed"
+)
+  .addOptionalParam(
+    "from",
+    "Address to send value from",
+    undefined,
+    types.string
+  )
+  .addPositionalParam(
```

Comment:
a844f8103e7204c8cc9c59a025878dc075d527d4

Reply:
What about `addParam` instead of `addPositionalParam`? I think explicitly setting a flag `--target-balance` is more descriptive than putting just an amount
```
npx hardhat ensure-eth-balance --target-balance "0.5 ether" 0x123
```
vs.
```
npx hardhat ensure-eth-balance "0.5 ether" 0x123
```

Reply:
f0376d7c2563e134f97f63ee06bd75445b373ed4